app/presenters/result_set_presenter.rb

```diff
@@ -49,13 +49,25 @@ def selected_filter_descriptions
     }.reject(&:empty?)
   end
 
+  def is_top_result?(score_a, score_b)
```

Comment:
I think this needs to be renamed too because the first results is always the "top" results when ordered by `-relevance`